app/pages/registration.tsx

```diff
@@ -33,6 +34,10 @@ class Registration extends Component<Props> {
       router
     } = this.props;
     const {ciipUserBySub, sub, givenName, familyName, email} = session || {};
+    let redirectURI = router.query.redirectTo;
+    // The redirectTo loses the query string information, which is needed when redirecting a certifier. The query string is still stored in pageProps
+    if (router?.components?.['/certify'])
+      redirectURI = `${router.query.redirectTo}?applicationId=${router.components['/certify'].props.pageProps.variables.applicationId}`;
```

Comment:
Looks like that's a bug in `default-layout`, where we need to use `router.asPath` instead of `router.route`